Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,256 @@
+                    }
+                }
+                .overlay(alignment: .center) {
+                    if isLoading {
+                        ProgressView("Loading…")
+                            .padding()
+                            .background(.ultraThickMaterial)
+                            .cornerRadius(10)
+                            .shadow(radius: 50)
+                    }
+                }
+                .toolbar {
+                    ToolbarItemGroup(placement: .bottomBar) {
+                        Button(action: {
+                            presentAlert = true
+                        }, label: {
+                            Text("Open Service")
+                        })
+                        Spacer()
+                        if !selection.isEmpty {
+                            Picker("Layers", selection: $selection) {
+                                ForEach(model.layerNames, id: \.self) { title in
+                                    Text(title)
+                                }
+                            }.task(id: selection) {
+                                model.update(for: selection)
+                                if let selection = model.selectedInfo {
+                                    do {
+                                        try await model.displayLayer(
+                                            with: selection,
+                                            proxy: mapProxy
+                                        )
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            }
+                            .pickerStyle(.automatic)
+                        }
+                    }
+                }
+                .alert("Load OGC API feature service", isPresented: $presentAlert, actions: {
+                    TextField("URL:", text: $userInput)
+                    Button("Load", action: {
+                        presentAlert = false
+                        isLoading = true
+                        Task {
+                            do {
+                                try await model.loadOGCFeatureData(
+                                    mapProxy: mapProxy,
+                                    url: URL(string: userInput)
+                                )
+                                selection = model.layerNames.first ?? ""
+                            } catch {
+                                self.error = error
+                            }
+                        }
+                    })
+                    Button("Cancel", role: .cancel, action: {
+                        presentAlert = false
+                    })
+                }, message: {
+                    Text("Please provide a URL to an OGC API feature service.")
+                })
+                .errorAlert(presentingError: $error)
```

Comment:
Overall review whether more logic can be moved to the model, the view is becoming crowded. Also review the model for redundancies.